cp3pt0-deployment/setup_singleton.sh

```diff
@@ -112,10 +124,10 @@ function install_cert_manager() {
     if [ $ENABLE_PRIVATE_CATALOG -eq 1 ]; then
         SOURCE_NS="ibm-cert-manager"
```

Comment:
@YCShen1010 

If it is a private CatalogSource, the CatalogSource will be in the same namespace as operator's.
Could we update the namespace here as well? Thank you!